src/Pond.tsx

```diff
@@ -65,26 +67,43 @@ export const Pond = ({
   loadComponent,
   onError,
   url,
-  onStoreConnectionClosed
+  onStoreConnectionClosed,
+  pondOptions
 }: PondProps) => {
   const [pond, setPond] = React.useState<PondType>()
+  const [errorStateComponent, setErrorStateComponent] = React.useState<JSX.Element | undefined>(
+    undefined
+  )
+
+  if (onError === undefined) {
+    console.warn(
+      'Not providing an onError handler is deprecated. Add a mechanism (e.g. reload) for handling connection problems with Actyx'
```

Comment:
why not require it in this version? deprecation is normally used the other way around, but in this case we’re plugging a hole (or rather: keeping users from shooting at their feet)

Reply:
it requires a major version bump. I like to avoid that.

Reply:
I think it is quite important to fix this hole — I don’t see a downside in bumping the major version.

Reply:
When we keep the current version, it might get updated to some projects automatically. 

They probably get the warning after the next `npm install`. 

The major should at least be protected. They have to update the package on purpose.

But we do not have to wait very long to go with the next version. Hopefully, we get some more things to change.